### ISSUE TITLE
Ability to store an image locally and distort/save it before completing upload

### ISSUE THREAD

**Issue by jywarren** [publiclab/mapknitter#340]
We could change our upload workflow so that when you drag in an image, it's stored in the browser, and we can save distortions, and the later sync it up to the online app when you get an internet connection AND/OR the upload completes.

**Comment by gauravano** [comment]
Nice idea!

**Comment by avsingh999** [comment]
Yes, great Idea @jywarren But How can be Implement this feature.

**Comment by jywarren** [comment]
Currently the workflow is:

1. upload image
2. hear back from server
3. instantiate the Leaflet.DistortableImage overlay object in JS
4. display to user

Here, we'd start by instantiating Leaflet.DistortableImage overlay object
in JS, as soon as possible. Then we'd start the upload process in the
background, but we'd want to show some kind of indication of uploading
status. What might that look like?
